itsy-bitsy-data-structures-ru_RU.js

```diff
@@ -1098,8 +1098,8 @@ class LinkedList {
  *       Heap, Binary Heap, Weak Heap, Binomial Heap, Fibonacci Heap, Leonardo
  *       Heap, 2-3 Heap, Soft Heap, Pairing Heap, Leftist Heap, Treap, ...
  *
- *     Trees:
- *       Trie, Radix Tree, Suffix Tree, Suffix Array, FM-index, B-trie, ...
```

Comment:
It's an original part of the main article, not changed by this translation:
https://github.com/jamiebuilds/itsy-bitsy-data-structures/blob/master/itsy-bitsy-data-structures.js#L1134